utils/frontend_utils.py

```
@@ -384,13 +446,18 @@ def find_model_directory(model_name: str):
     static_dir = 'static'
     static_dir_real = os.path.realpath(static_dir)
     static_model_dir = os.path.join(static_dir, model_name)
+    static_dir = 'static'
+    static_dir_real = os.path.realpath(static_dir)
+    static_model_dir = os.path.join(static_dir, model_name)
     if os.path.exists(static_model_dir):
         static_model_dir_real = os.path.realpath(static_model_dir)
         if os.path.commonpath([static_model_dir_real, static_dir_real]) == static_dir_real:
             return (static_model_dir, '/static')
     
     # 如果都不存在，返回None
```

Ruff (0.14.11)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)